src/builtins/sci-embodied/README.md

```diff
@@ -0,0 +1,111 @@
+```
+
+Where:
+
+- `device/emissions-embodied` = Total embodied emissions; the sum of Life Cycle Assessment (LCA) emissions for the component.
+
+- `timeShare` = Time-share; the share of the total life span of the hardware reserved for use by an application.
+
+  - `timeShare` is calculated as `duration/'device/expected-lifespan'`, where:
+    - `duration` = the length of time the hardware is reserved for use by the software.
+    - `device/expected-lifespan` = Expected lifespan: the length of time, in seconds, between a component's manufacture and its disposal.
+
+- `resourceShare` = Resource-share; the share of the total available resources of the hardware reserved for use by an application.
+  - `resourceShare` is calculated as `resources-reserved/resources-total`, where:
+    - `resources-reserved` = Resources reserved; the number of resources reserved for use by the software.
+    - `resources-total` = Total Resources; the total number of resources available.
+
+## Implementation
+
+IF implements the plugin based on the logic described above. To run the plugin, you must first create an instance of `SciEmbodied`. Then, you can call `execute()` to return `m`.
+
+## Usage
+
+The following snippet demonstrates how to call the `sci-embodied` plugin from Typescript.
+
+```typescript
+import {SciEmbodied} from '@grnsft/if-plugins';
+
+const sciEmbodied = SciEmbodied();
+const results = await sciEmbodied.execute([
+  {
+    'device/emissions-embodied': 200, // in gCO2e for total resource units
+    duration: 60 * 60 * 24 * 30, // time reserved in seconds, can point to another field "duration"
+    'device/expected-lifespan': 60 * 60 * 24 * 365 * 4, // lifespan in seconds (4 years)
+    'resources-reserved': 1, // resource units reserved / used
+    'resources-total': 1, // total resource units available
+  },
+]);
+```
+
+## Example manifest
+
+IF users will typically call the plugin as part of a pipeline defined in a `manifest` file. In this case, instantiating the plugin is handled by `ie` and does not have to be done explicitly by the user. The following is an example `manifest` that calls `sci-embodied`:
+
+```yaml
+name: sci-embodied
+description: simple demo invoking sci-embodied
+tags:
+initialize:
+  outputs:
+    - yaml
+  plugins:
+    sci-embodied:
+      method: SciEmbodied
+      path: '@grnsft/if-plugins'
+tree:
+  children:
+    child:
+      pipeline:
+        - sci-embodied # duration & config -> embodied
+      defaults:
+        device/emissions-embodied: 1533.120 # gCO2eq
+        device/expected-lifespan: 3 # 3 years in seconds
+        resources-reserved: 1
+        resources-total: 8
+      inputs:
+        - timestamp: 2023-07-06T00:00
+          duration: 3600
+```
+
+You can run this example `manifest` by executing the following command from the project root:
+
+```sh
+npm i -g @grnsft/if
+npm i -g @grnsft/if-plugins
+ie --manifest ./examples/manifests/test/sci-embodied.yml --output ./examples/outputs/sci-embodied.yml
```

Comment:
the manifest path is changed, please, update

Reply:
```suggestion
ie --manifest manifests/plugins/sci-embodied.yml --output manifests/outputs/sci-embodied.yml
```